src/__tests__/a11yAPI.test.js

```diff
@@ -62,10 +62,20 @@ function Section() {
   );
 }
 
-test('getByA11yLabel, queryByA11yLabel, findByA11yLabel', async () => {
-  const { getByA11yLabel, queryByA11yLabel, findByA11yLabel } = render(
-    <Section />
-  );
+test('getByA11yLabel, getByLabelText, queryByA11yLabel, queryByLabelText, findByA11yLabel, findByLabelText', async () => {
+  const {
+    getByA11yLabel,
+    getByLabelText,
+    queryByA11yLabel,
+    queryByLabelText,
+    findByA11yLabel,
+    findByLabelText,
+  } = render(<Section />);
+
+  // Assert that query aliases are referencing the same function
+  expect(getByA11yLabel).toBe(getByLabelText);
+  expect(queryByA11yLabel).toBe(queryByLabelText);
+  expect(findByA11yLabel).toBe(findByLabelText);
```

Comment:
This looks way better now. Can we extract all reference equality checks to a separate test? Something like:
```js
test('a11y queries have aliases', () => {
const {
     getByA11yLabel,
     getByLabelText,
     queryByA11yLabel,
     queryByLabelText,
     findByA11yLabel,
     findByLabelText,
     ... // "query all by" here
   } = render(<Section />);

    // Assert that query aliases are referencing the same function
   expect(getByA11yLabel).toBe(getByLabelText);
   expect(queryByA11yLabel).toBe(queryByLabelText);
   expect(findByA11yLabel).toBe(findByLabelText);
})
```
this way we don't need to alter existing tests

Reply:
Even better! I like this. I'll push an update shortly 👍